paparazzi/src/test/java/app/cash/paparazzi/HtmlReportWriterTest.kt

```diff
@@ -36,7 +36,8 @@ class HtmlReportWriterTest {
   val snapshotRoot: TemporaryFolder = TemporaryFolder()
 
   private val anyImage = BufferedImage(1, 1, BufferedImage.TYPE_INT_ARGB)
-  private val anyImageHash = "9069ca78e7450a285173431b3e52c5c25299e473"
+  private val anyImageHash = "2ef6a6e795a2b8cffabefb6d9a2066a183c8e3b6"
```

Comment:
any idea why `anyImageHash` changes?

Reply:
Before snapshot specified the image hash `val original = File(imagesDirectory, "${hashes[0]}.png")` but now both snapshots and gifs share a common path / hashing in close `val hashedFile = File(snapshotDir, "${hash(hashes)}.png")`